.github/workflows/develop-pr.yml

```diff
@@ -0,0 +1,81 @@
+name: docker develop build and test only
+
+on:
```

Comment:
You can further enhance this by adding:
```
on:
  pull_request:
    branches:
      - main
```

this will allow this workflow to only trigger for PR requests against `main` branch.

Reply:
done